playbooks/operationalizing/add_node.adoc

```diff
@@ -0,0 +1,120 @@
+osm_default_subdomain=apps.example.com
+
+openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/openshift-passwd'}]
+
+# host group for masters
+[masters]
+master.ose.example.com 
+
+# host group for nodes
+[nodes]
+master.ose.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+node1.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'north'}"
+node2.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'south'}"
+----
+
+Next we would like to add in two more nodes. First, the *openshift_pkg_version* and *g_new_node_hosts* variables have to be added to the [OSEv3:vars] section. The *openshift_pkg_version* variable should be populated with the correct version based on the operational OpenShift environment. There are many ways the version string can be obtained, but one option is to login to the CLI of the master and use the "rpm" command:
+
+    # rpm -q atomic-openshift-master
+    atomic-openshift-master-3.1.0.4-1.git.15.5e061c3.el7aos.x86_64
+
+Based on the example above, the version string is what is between "atomic-openshift-master" and ".x86_64" - i.e.: "-3.1.0.4-1.git.15.5e061c3.el7aos"
+
+**__NOTE__** _the leading '-' must be included_
+
+
+Lastly, the new nodes need to be added to the *g_new_node_hosts* variable as well as in the [nodes] host group with the appropriate parameters.
+
+The resulting inventory file then becomes:
+
+[source]
+----
+[OSEv3:children]
+masters
+nodes
+
+[OSEv3:vars]
+deployment_type=openshift-enterprise
+ansible_ssh_user=root
+osm_default_subdomain=apps.example.com
+openshift_pkg_version=-3.1.0.4-1.git.15.5e061c3.el7aos <1>
+g_new_node_hosts=[node3.ose.example.com, node4.ose.example.com] <2>
+
+openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/openshift-passwd'}]
+
+# host group for masters
+[masters]
+master.ose.example.com
+
+# host group for nodes
+[nodes]
+master.ose.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+node1.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'north'}"
+node2.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'south'}"
+node3.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'north'}" <3>
+node4.ose.example.com openshift_node_labels="{'region': 'primary', 'zone': 'south'}" <4>
+----
+<1> OpenShift version string for the environment
+<2> New OpenShift nodes to be added
+<3> New node listed in the [nodes] host group
+<4> Same as 3., new node listed in the [nodes] host group
+
+Next, run the ansible-playbook for scaleup with the inventory file above:
+
+    ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/scaleup.yml
+
+== Post ScaleUP tasks
```

Comment:
A link to the [Verification Guide](https://github.com/rhtconsulting/openshift-playbooks/blob/master/playbooks/installation/install_validation.adoc) would also be helpful